### DISCUSSION TITLE
Python Wrapper: Passing Torch Tensor to inverseDynamics()

### DISCUSSION THREAD

**Discussion by KyunHwan** [rbdl/rbdl#74] [Q&A]
At the moment the function seems to only take in numpy arrays and output a numpy array, but I was wondering if there's an easy way to allow the function to take in torch tensors as input (ie. q, qdot, qddot) and return a torch tensor. I'm asking this because I'm trying to use q, qdot, and qddot as parameters for torch optimization and wanted to track the calculations.

**Comment by ju6ge** [comment]
There are no plans to support torch directly. You probably can wrap the functions you need with something that does a conversion though.
